OstapChyzho/HW7/Task3 .py

```diff
@@ -0,0 +1,16 @@
+def character_positions(wrd):
+    """
+    calculate the number of characters
```

Comment:
the task condition was different
look at this implementation

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result